src/jenkins/BuildManifest.groovy

```
@@ -27,10 +27,29 @@ class BuildManifest implements Serializable {
         }
     }
 
+    class Components implements Serializable {
+        String dist
+
+        Components(Map data) {
+            this.dist = data.artifacts.dist
+        }
+
+        String getDistPackageLocation() {
+            try {
+                return this.dist[0]
+            } catch (Exception e) {
```

This just swallows the error. What we want is a better error.